packages/graphql/package.json

```diff
@@ -47,5 +49,30 @@
     "src",
     "schemas"
   ],
-  "types": "src/index.ts"
+  "types": "src/index.ts",
+  "nx": {
+    "targets": {
+      "nexus-build": {
+        "outputs": [
+          "{projectRoot}/src/gen/nxs.gen.ts",
+          "{projectRoot}/schemas/schema.graphql"
+        ]
+      },
+      "build": {
+        "dependsOn": [
+          "nexus-build",
```

Comment:
Not ideal - we should split `graphql-codegen.yml` into different yaml files, one for each package that generates types, and then have each package generate its own gql type files